### ISSUE TITLE
EIPIP Meeting 108 

### ISSUE THREAD

**Issue by poojaranjan** [ethcatherders/EIPIP#350]
# Date and Time

Aug 21, 2024 at 14:00 UTC 

# Location

Zoom: TBA in the Discord #eip-editing channel

YouTube Recording: [EIPIP Meetings](https://www.youtube.com/playlist?list=PL4cwHXAawZxpLrRIkDlBjDUUrGgF91pQw)

# Agenda

## 1. Discuss Open Issues/PRs, and other topics

- [x] disabling the stale bot Ref: [here](https://github.com/ethcatherders/EIPIP/issues/350#issuecomment-2298781447)
- [x] FORK_TIMESTAMP variable that is not specified for Dencun - https://eips.ethereum.org/EIPS/eip-7045

### Edit to `Final` Proposals
- [ ] https://github.com/ethereum/EIPs/pull/8817

### Update to EIP-1
_TBA_

#### Misc Issue/ requires attention?
_TBA_

 
### Call for Input

Call For Input | Status | Inclination/Result | Deadline
-- | -- | -- | --
[343](https://github.com/ethcatherders/EIPIP/issues/343) | Closed| Merged |July 28, 2024
[345](https://github.com/ethcatherders/EIPIP/issues/345) | Closed | Delete  |Aug 02, 2024
[348](https://github.com/ethcatherders/EIPIP/issues/348) | Closed | Merged |Aug 17, 2024
[349](https://github.com/ethcatherders/EIPIP/issues/349) | Open |  |Aug 16, 2024

## 2. Other discussions and updates from past meetings
_TBA_

## 3. EIPs Insight - Monthly EIPs status reporting.

Repo | Issues | PRs | Total
-- | -- | -- | --
EIP |  3 | 50  | 53
ERC | 6  | 71  |77
RIP |  3 | 14 | 17

- [EIPsInsight.com](https://eipsinsight.com/insight/2024/7)
- [Hackmd](https://hackmd.io/@poojaranjan/EthereumImprovementProposalsInsight)

## 4. [EIP Editing Office Hour](https://hackmd.io/@poojaranjan/EIPEditingOfficeHour)
- Agenda [43](https://github.com/ethcatherders/EIPIP/issues/355) Video [42](https://youtu.be/2NxR5gmKZRs)

## 5. Review action items from earlier meetings
- [Summary meeting 107](https://github.com/ethcatherders/EIPIP/issues/346#issuecomment-2241093077)

## Next Meeting date & time - Sep 18, 2024 (3rd Wed of the Month) at 14:00 UTC

**Comment by smartprogrammer93** [comment]
Hey team,
I found another instance of `FORK_TIMESTAMP` variable that is not specified for Dencun:
https://eips.ethereum.org/EIPS/eip-7045
Not sure if you want to act on it similar to 4788 or leave it. Thought that since there is precedence to fixing such issues, that we can fix them all.

**Comment by poojaranjan** [comment]
@smartprogrammer93 
As @SamWilsn mentioned we can take it case by case. 
Most likely, a _Call For Input_ will be created if we receive a Pull Request to update the EIP. 
The final decision will be of the editors. However, I'd be in favor of merging the PR.

**Comment by SamWilsn** [comment]
Can we add "disabling the stale bot" to the agenda?

It's just annoying and doesn't take into account whether an issue/PR is waiting on author or editor.

**Comment by viwinkumarpadala** [comment]
Hey team,
I want to provide a demo for our website: [EIPsInsight](https://eipsinsight.com/ )
We have created tooling designed to provide clear, visual insights into the activity of Ethereum Improvement Proposal (EIP), Ethereum Request for Comments (ERCs), and Rollup Improvement Proposals (RIPs) over a specified period. Data provided can be used for tracking the progress all the proposals. We have also added a few tools that can be useful for keeping track of PRs and Reviews.

**Comment by poojaranjan** [comment]
## Summary
### 1. Discuss Open Issues/PRs, and other topics
#### disabling the stale bot 
- @SamWilsn proposed to disable the bot as it is closing PRs before the review
- @Lightclient suggested to edit/update the bot instead of disabling it completely
- Todos
  - @SamWilsn will create PRs to 
  - Update the bot to continue providing warnings with edits to 6 months
  - Send a message to provide 1 week warning before closing
  The ideal expectation will be to have the bot close PR waiting  for the author and not the editors

Ref: [PR for EIP](https://github.com/ethereum/EIPs/pull/8820) and [PR for ERC]https://github.com/ethereum/ERCs/pull/600)

 #### FORK_TIMESTAMP variable that is not specified for Dencun - https://eips.ethereum.org/EIPS/eip-7045
- Added to the agenda by @smartprogrammer93 
- @SamWilsn will like to consider the PR
- @poojaranjan will work with @djrtwo to get the PR up

### Edit to Final Proposals
#### https://github.com/ethereum/EIPs/pull/8817
- @SamWilsn created this PR to fix a small typo
- Need 3 EIP Editors confirmation to merge the PR 

### Call For Input
#### https://github.com/ethcatherders/EIPIP/issues/349
- This is the deadline. There are only two comments, one in favor and one in against.
- It would require more Editors comment to make a decision.

### 3. EIP Insight
- @poojaranjan shared the demo for tooling added
  - [Status Timeline](https://eipsinsight.com/eips/eip-1559): Pictorial representation of the change of status with number of days in each status
  - [PRs & Issues](https://eipsinsight.com/openClosePrs#) demo shared. 
    - Sam Wilson suggested that the chart can be updated to Bar or Line chart against the timeline.
  - [Editors Review](https://eipsinsight.com/editorsReview) is breaking with date parameter. The team will take a look fix it. 
    - The chart against timeline will be good to have.
- Todo for EIPsInsight team
  - Add Network Upgrade  to EIP page
  - Add charts to include against the timeline
@SamWilsn also recommended adding [EIPsInsight website link](https://eipsinsight.com) to the [EIPs Website](https://eips.ethereum.org/).

## The [next meeting](https://github.com/ethcatherders/EIPIP/issues/356) is scheduled for Sep 18, 2024 at 14:00 UTC

**Comment by poojaranjan** [comment]
Closing in favor #356
